### DISCUSSION TITLE
Android & iOS maps not synced

### DISCUSSION THREAD

**Discussion by davidru85** [organicmaps/organicmaps#8174] [Q&A] [answered]
Hi
Maybe is something already documented but I haven't found anything about it.
I have an Android phone and an iPad and I've just realised Android and iOS apps, have different map versions.

As you can see in the screenshots below (those are in Spanish but I can translate it to you), maps in the Android version are from 26th March and maps in the iPadOS version are from 29th April.
Is this the normal behaviour or this is some kind of issue?
Thanks

![Screenshot_20240514-173811_Original](https://github.com/organicmaps/organicmaps/assets/472324/8ad521ad-cdb1-4e90-8d6c-d4f066c41f74)
![IMG_1989](https://github.com/organicmaps/organicmaps/assets/472324/be6f9cd0-a11d-4d86-a838-325d4821f134)

**Comment by Misalf-git** [comment]
Update the apps, then maps updates will become available.

**Reply by davidru85** [reply]
Both apps are updated, but thanks. Now I see it's just a matter of app version.

**Comment by biodranik** [comment, marked as answer]
This is not a bug. Map updates are tied to app updates at the moment.

**Reply by davidru85** [reply]
I though app versions and map version didn't need to be "synced"
Thanks for the quick answer!
